charts/tfy-karpenter-config/values.yaml

```diff
@@ -54,14 +54,15 @@ karpenter:
     ## @param karpenter.defaultNodePool.capacityTypes Capacity types for the default node pool
     capacityTypes:
       - spot
+      - on-demand
     ## @param karpenter.defaultNodePool.zones Zones for the default node pool
     zones: []
     ## @param karpenter.defaultNodePool.architectures Architectures for the default node pool
     architectures:
       - amd64
     ## @param karpenter.defaultNodePool.instanceFamilies.notAllowed Not allowed instance families for the default node pool
     instanceFamilies:
-      notAllowed: ["t3", "t2", "t3a", "p2", "p3", "p4d", "p4de", "p5", "g4dn", "g5", "g4ad", "inf1", "inf2", "trn1", "trn1n"]
+      notAllowed: ["t3", "t2", "t3a", "p2", "p3", "p4d", "p4de", "p5", "g4dn", "g5", "c1", "cc1", "cc2", "cg1", "cg2", "cr1", "g1", "g2", "hi1", "hs1", "m1", "m2", "m3", "g4ad", "inf1", "inf2", "trn1", "trn1n"]
```

Comment:
I think c1 and these were removed because they have really less IP address available and frequently run into ip exhaustion issue